authority-portal-frontend/src/app/core/api/fake-backend/impl/fake-connectors.ts

```diff
@@ -30,6 +31,20 @@ import {fakeEnv} from './fake-environments';
 import {TEST_ORGANIZATIONS} from './fake-organizations';
 import {getUserInfo} from './fake-users';
 
+const ACTIVATE_RANDOM_STATUSES: boolean = false;
+
+const getRandomStatus = (): ConnectorStatusDto => {
+  const statuses: ConnectorStatusDto[] = ['ONLINE', 'OFFLINE', 'INIT'];
+  return statuses[Math.floor(Math.random() * statuses.length)];
+};
+
+if (ACTIVATE_RANDOM_STATUSES)
+  setInterval(() => {
+    TEST_CONNECTORS.forEach((c) => {
+      c.status = getRandomStatus();
+    });
+  }, 20000);
```

Comment:
you could add logic to created/launched connectors that updates their status after 5 seconds